res/css/_components.scss

```diff
@@ -132,6 +132,7 @@
 @import "./views/elements/_EditableItemList.scss";
 @import "./views/elements/_ErrorBoundary.scss";
 @import "./views/elements/_EventListSummary.scss";
+@import "./views/elements/_EventTilePreview.scss";
```

Comment:
Please make sure `yarn rethemendex`

Reply:
`yarn rethemendex` runs without an error but also doesn't change any files. so I expect everything is fine?

Reply:
Yeah, everything is fine then :)